js/src/lib/interfaces/Types.ts

```diff
@@ -459,6 +467,10 @@ export const PIPELINE_DATA = ensureRecordOfPipelines({
 				type: "tabular-multi-label-classification",
 				name: "Tabular Multi Label Classification",
 			},
+			{
+				type: "tabular-binary-classification",
```

Comment:
This doesn't seem to be a task in autotrain ? it makes sense to have it here though anyway :)

Reply:
Yes, it is defined here: https://github.com/huggingface/autonlp-backend/blob/918474d3e0ee914f7ebc50f6d89e20397eacd95e/database/database/enums.py#L120